### PR TITLE
Adds schema for domain config

### PR DESCRIPTION
**Pull request by davidporter-id-au**
Adds domain isolation group schema

### CODE REVIEW

File: schema/cassandra/cadence/versioned/v0.35/partition_config.cql
Comment (Shaddoll) [inline schema/cassandra/cadence/versioned/v0.35/partition_config.cql:1]: could you also update `schema.cql`?

Comment (davidporter-id-au) [reply schema/cassandra/cadence/versioned/v0.35/partition_config.cql:1]: Little bit confused about this, if I add it to the base schema.cql, won't this cause problems

**Review by Shaddoll — APPROVED** [review]

File: schema/cassandra/cadence/schema.cql [hunk 1]
```diff
@@ -289,6 +294,7 @@ CREATE TYPE domain_config (
   visibility_archival_uri text,
   bad_binaries    blob,
   bad_binaries_encoding blob,
+  isolation_groups frozen<list<isolation_group_partition>>;
```
Comment (Shaddoll) [inline schema/cassandra/cadence/schema.cql:297]: nit: ";" should be ","

**Review by davidporter-id-au — COMMENTED** [review]
fix typo

**Pull request merged**